kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3Storage.java

```diff
@@ -218,6 +225,18 @@ public void delete(String name) {
   @Override
   public void close() {}
 
+  public void addTags(String fileName, Map<String, String> tags)
+          throws SdkClientException {
+    List<Tag> newTagList = new ArrayList<>();
+    for (Map.Entry<String, String> entry : tags.entrySet()) {
+      newTagList.add(new Tag(entry.getKey(), entry.getValue()));
+    }
+    s3.setObjectTagging(new SetObjectTaggingRequest(
+            this.bucketName,
```

Comment:
nit: I also now realize that your tab size in some cases is equal to 4 spaces. 

In this project it should be: 
Tab size: 2
Indent: 2
Continuation indent: 4
(these are the settings in IntelliJ, similar should be set to other envs)